exist-parent/pom.xml

```diff
@@ -682,7 +682,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-scm-plugin</artifactId>
-                    <version>1.11.12</version>
+                    <version>1.11.2</version>
```

Comment:
looks like a downgrade?

Reply:
It was a typo, there is no such version as `1.11.12`, latest available version is `1.11.2`